method-checklist.md

```diff
@@ -0,0 +1,39 @@
+# Method publishing checklist
+
+Please follow the checklist to ensure your content is ready for publishing on the [Methods Hub portal](https://methodshub.gesis.org/). *Note: the content submitted for publishing is held for review to crosscheck all requirements before publishing.*
+
+Methods that have [established communities](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#13alternative-for-established-methods) or are published by [trusted third parties](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#12trusted-third-party-review-bodies) can be considered as special case (bypassing some of the documentation requirements) for publishing through Methods Hub. *Please consult Methods Hub Team first by [email](methodshub@gesis.org) for further details.*
+
+## Checklist
+**Section 1:  Preconditions as openness criteria (ref. [method preconditions](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#3-method-preconditions))**
+- [ ] Developed in open-sourced programming language e.g., Python, R 
+- [ ] Publicly accessible script 
+- [ ] Open license  
+
+**Section 2:  Scope criteria as relevance to social science (ref. [method scope](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#4-scoping-criteria))**
+- [ ] Establishing social science relevance through use case, research question or citing relevant social science publications 
+- [ ] The method is applicable to Digital Behavioral Data (DBD) (ideally [GESIS DBD](https://www.gesis.org/en/institute/about-us/digital-behavioral-data)), also demonstrated through sample data in the repo 
```

Comment:
Avoid clipping ("repo") and prefer the full word ("Git Repository" or "Project").